src/test/css/nodes.test.ts

```diff
@@ -102,9 +114,16 @@ suite('CSS - Nodes', () => {
 		return node;
 	}
 
+	function stylesheet(input: string): nodes.Stylesheet {
+		let parser = new Parser();
+		let node = parser.internalParse(input, parser._parseStylesheet)!;
+		return node;
+	}
+
 	test('RuleSet', function () {
-		assertNodes(ruleset, 'selector { prop: value }', 'ruleset,...,selector,...,declaration,...,property,...,expression');
-		assertNodes(ruleset, 'selector { prop; }', 'ruleset,...,selector,...,selector');
```

Comment:
This was a false positive from the old test helper.
There is only one selector in this source, but the test still passed.